oar/cli/cmd_check_blocking_sec_alerts.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+import click
+
+from oar.core.advisory import AdvisoryManager
+from oar.core.const import *
+from oar.core.worksheet import WorksheetManager
+
+logger = logging.getLogger(__name__)
+
+@click.command()
+@click.option("--check-only", is_flag=True, default=False, help="Only check for blocking alerts without updating worksheet")
+@click.pass_context
+def check_blocking_sec_alerts(ctx, check_only):
+    """
+    Check for blocking security alerts across all advisories and update Others column with results
+    """
+    # get config store from context
+    cs = ctx.obj["cs"]
+
+    try:
+        # Get existing report
+        wm = WorksheetManager(cs)
+        report = wm.get_test_report()
+
+
+
+        # Get all advisories and check them directly
+        am = AdvisoryManager(cs)
+        advisories = am.get_advisories()
+
+        # Initialize result structure
+        has_blocking = False
+        details = {
+            "checked_advisories": [],
+            "blocking_advisories": [],
+            "errors": []
+        }
+
+        # Check each advisory using existing has_blocking_secruity_alert method
+        for advisory in advisories:
+            advisory_info = {
```

Comment:
`advisory_info` is useless, all the attribute can be retrieved from `Advisory` object.